packages/node-terra/src/indexer/apiterra.service.ts

```diff
@@ -76,10 +89,54 @@ export class TerraClient {
     return this.baseApi.tendermint.blockInfo(height, this.params);
   }
 
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async blockInfoMantlemint(height?: number): Promise<BlockInfo> {
+    const config = {
+      method: 'get',
+      url: `${this.mantlemintURL}/index/blocks/${height}`,
+    };
+    return axios(config)
+      .then((d) => d.data)
+      .catch((e) => {
+        throw e;
+      });
```

Comment:
We can use async await for these. (can also remove the eslint disable)

```suggestion
    const { data } = await axios(config);
    return data;
```